dhcpv6/ztpv6/parse_vendor_options.go

```diff
@@ -33,8 +33,16 @@ func ParseVendorData(packet dhcpv6.DHCPv6) (*VendorData, error) {
 	vData := []string{}
 
 	if opt17 != nil {
-		vo := opt17.(*dhcpv6.OptVendorOpts).VendorOpts
-		for _, opt := range vo {
+		vendorOptsOption := opt17.(*dhcpv6.OptVendorOpts)
```

Comment:
No strong opinion here: Is the new `vendorOptsOption` variable clearer than before?

Reply:
Distinguishing between a sub-option and the actual option was hard (which one is opt and which one is vo?)